### DISCUSSION TITLE
Sniffing TP Link/Kasa devices?

### DISCUSSION THREAD

**Discussion by Gibber** [Koenkk/zigbee2mqtt#7060] [Devices]
Has anyone successfully sniffed any ZigBee messages of TP link/Kasa devices? I have a CS100 contact sensor and an MS100 motion sensor. From pairing them with a TP Link SR20 router I know that the MS100 has a motion sensitivity setting option but I cannot figure out how to configure this. 
I have a spare CC2531 dongle that I have used to sniff ZigBee messages between a Raspberry Pi with another CC2531 and the MS100 and it works like a charm using the TI firmware and software. 
When I try and sniff traffic between the TP Link SR20 and the MS100 I get nothing at all. I have tried several RF channels but no help.

Any ideas? Is there another parameter than RF channel that I should try?

**Comment by Gibber** [comment]
I finally found the answer. Not sure how I missed it before but through trial and error I found the TP Link/Kasa system uses ZigBee channel 17. Once I found that I got lots of packets. 
Now I just need to sort out what they mean!

**Comment by alexsahka** [comment]
Any progress on the TP-Link MS100? Is it possible to use with Z2M?

**Reply by Gibber** [reply]
Hi,
Nope could not get it to work so I switched to the ZigBee binding and that was better but did not work with my Ikea Tradfri devices so I ended up switching to Home Assistant and the ZigBee integration and everything works great!
